stglib/eofe.py

```diff
@@ -0,0 +1,632 @@
+from __future__ import division, print_function
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+import seawater as sw
```

Comment:
If you are going to rely on the seawater toolbox we need to add it to the requirements.

Reply:
Added "seawater" to requirements.yml and .txt files, in alphabetical order in list.